src/pages/User/components/PullRequests/index.js

```diff
@@ -63,40 +96,25 @@ class PullRequests extends Component {
   fetchPullRequests = async () => {
     try {
       const username = this.props.username;
-      const apiUrl = [
-        `https://api.github.com/search/issues?q=author:${username}+is:pr+created:2019-10-01..2019-10-31`,
-        `https://api.github.com/search/users?q=user:${username}`
-      ];
-
-      this.setState({
-        loading: true
-      });
-
-      const allResponses = apiUrl.map(url =>
-        fetch(url, {
-          headers: {
-            Authorization: `token ${GITHUB_TOKEN}`
-          }
-        })
-          .then(response => response.json())
-          .catch(error =>
-            this.setState({
-              loading: false,
-              error
-            })
-          )
-      );
-
-      const [data, userDetail] = await Promise.all(allResponses);
+      const { data, userDetail, isOrgMember } = await fetchUserInfo(username);
       const count = this.counterOtherRepos(data, userDetail);
 
-      this.setState({
-        data: this.getValidPullRequests(data),
-        userDetail,
-        loading: false,
-        otherReposCount: count,
-        error: null
-      });
+      if (!isOrgMember) {
+        this.setState({
```

Comment:
Extract these to separate methods with names reflecting what they'd do eg: `showNotAMemberModal`.